jest.config.ts

```diff
@@ -13,6 +13,7 @@ const reactMarkdownEsModules = [
   "micromark",
   "property-information",
   "react-markdown",
+  "react-modal",
```

Comment:
I think this belongs on line 29 similar to "@buoysoftware/anchor-.+". This array sees to cover the peer dependencies of react markdown

Reply:
Makes sense!